lib/carp_study_app.dart

```diff
@@ -118,9 +128,9 @@ class CarpStudyAppState extends State<CarpStudyApp> {
       GoRoute(
         path: InformedConsentPage.route,
         parentNavigatorKey: _rootNavigatorKey,
-        redirect: (context, state) => bloc.hasInformedConsentBeenAccepted
-            ? firstRoute
-            : (bloc.studyId == null ? InvitationListPage.route : null),
+        // redirect: (context, state) => bloc.hasInformedConsentBeenAccepted
+        //     ? firstRoute
+        //     : (bloc.studyId == null ? InvitationListPage.route : null),
```

Comment:
what happens if the user gets redirected to the ic page through deep linking for example, do they have to accept it again?

Reply:
I would say yes - the reason to navigate to the IC is to show it to the user (again). This is actually a missing features right now, that the user should be able to go back and see the IC - e.g. from the setting menu.